tests/playwright/sanity/core/app/modules/onboarding/onboarding.test.js

```diff
@@ -38,26 +38,30 @@ test.describe( 'On boarding', async () => {
 	 * text, And that clicking on it opens the popup to create an account in my.elementor.com
 	 */
 	test( 'Onboarding Create Account Popup Open', async ( { page } ) => {
+		// Arrange.
+		// Make sure Library is not connected
+		await page.goto( '/wp-admin/admin.php?page=elementor-connect' );
+		const resetButton = await page.waitForSelector( 'a[href*="elementor-connect&app=library&action=reset"' );
+		await resetButton.click();
+		await page.waitForLoadState( 'domcontentloaded' );
+
```

Comment:
Maybe use a different approach?

```suggestion
		await page.evaluate( () => {
		elementorAppConfig.onboarding.isLibraryConnected = false;
		} );

```